code_challenges/code_wars/zip_with.py

```diff
@@ -0,0 +1,24 @@
+from operator import add, sub, mul
+
+
+class ZipWith:
+    def __init__(self, func, array1, array2):
+        self.func = func
+        self.array1 = array1
+        self.array2 = array2
+
+    def zip_with(self):
+        return [self.func(a, b) for a, b in zip(self.array1, self.array2)]
```

Comment:
Part of the purpose of classes is reusability.  This class should be instantiated with a zip function, but be able to zip any 2 arrays.